src/utils/platform.ts

```diff
@@ -1,3 +1,7 @@
+export function isChrome() {
+    return navigator.userAgent.includes('Chrome');
```

Comment:
It should also work for Chromium browsers. See `getChromeVersion` function.

Reply:
It works for chromium browser Google have turned Chromium almost into chrome don't worry
![](https://i.imgur.com/84TKvhf.png)